library/ix-dev/community/adguard-home/questions.yaml

```diff
@@ -0,0 +1,181 @@
+groups:
+  - name: AdGuard Home Configuration
+    description: Configure AdGuard Home
+  # - name: User and Group Configuration
+  #   description: Configure User and Group for AdGuard Home
+  - name: Network Configuration
+    description: Configure Network for AdGuard Home
+  - name: Storage Configuration
+    description: Configure Storage for AdGuard Home
+  - name: Resources Configuration
+    description: Configure Resources for AdGuard Home
+
+portals:
+  web_portal:
+    protocols:
+      - "$kubernetes-resource_configmap_portal_protocol"
+    host:
+      - "$kubernetes-resource_configmap_portal_host"
+    ports:
+      - "$kubernetes-resource_configmap_portal_port"
+    path: "$kubernetes-resource_configmap_portal_path"
+
+questions:
+  # - variable: adguardRunAs
```

Comment:
Do we need this?

Reply:
```
# FIXME: It might be able to run rootless, probably blocked by:
# https://github.com/AdguardTeam/AdGuardHome/issues/4681
```
Don't know yet if when the issue above is solved it will be able to run as arbitrary user.